src/components/DocumentationTopic/Hierarchy.vue

```diff
@@ -178,15 +194,33 @@ export default {
 
 .hierarchy {
   justify-content: flex-start;
+  align-items: flex-start;
+  margin: 0;
   min-width: 0;
-  margin-right: 80px;
   @include nav-in-breakpoint() {
-    margin-right: 0;
+    margin: 0;
   }
 
   // make sure the root-hierarchy has a limit as well
   .root-hierarchy .item {
     @include truncate(10rem);
   }
 }
+
+// Applies colors to any link inside the nav
+:deep(.nav-menu-link) {
+  color: var(--color-nav-link-color);
+
+  &:hover {
+    color: var(--color-nav-link-color-hover);
```

Comment:
Some of the links are underlined and blue without hover, some are not. I think this is not expected? 
<img width="378" alt="image" src="https://github.com/apple/swift-docc-render/assets/87735557/e35e9a45-afb3-4771-9fa1-37dcf8014550">
<img width="493" alt="image" src="https://github.com/apple/swift-docc-render/assets/87735557/7454b448-070c-4472-8028-2933a98e6214">

Reply:
Fixed here https://github.com/apple/swift-docc-render/pull/772/commits/437d7554d823495fceec1a7787689f18ad197d97